index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pizzeria Site</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="js/index.js" type="module" defer></script>
```

Comment:
`type="module" defer>` przez to nie działa aplikacja. Dlaczego tego użyłaś?

Reply:
początkowo myślałam, żeby podzielić na moduły, ale finalnie tego nie zrobiłam i zapomniałam usunąć type="module".
Defer dodałam, zamiast umieszczania skryptu na dole strony, aby najpierw załadował się DOM.